library/swift/src/filters/FilterHeadersStatus.swift

```diff
@@ -9,5 +9,5 @@ public enum FilterHeadersStatus<T: Headers>: Equatable {
   /// Returning `continue` from `onRequestData()`/`onResponseData()` or calling
   /// `continueRequest()`/`continueResponse()` MUST occur when continued filter iteration is
```

Comment:
These docstrings are outdated (can you grep for any references to these functions and update please?)